debian/bullseye/foreman/foreman-plugin.mk.in

```diff
@@ -1,16 +1,40 @@
 BUNDLE = @BUNDLE@
 
 build:
-	cp cache/* /usr/share/foreman/vendor/cache/
-	cp $(PLUGIN).rb /usr/share/foreman/bundler.d/
-	cd /usr/share/foreman && ( \
+	# prepare a local foreman installation based on packages
+	mkdir -p ./usr/share
+	cp -r /usr/share/foreman ./usr/share || true
+
+	# clean up possibly existing state we copied
+	mkdir -p ./var/lib/foreman
+	cp -r /var/lib/foreman/db ./var/lib/foreman || true
+	unlink ./usr/share/foreman/db
+	mkdir ./usr/share/foreman/db
```

Comment:
Unrelated, but now that we've dropped SQLite I wonder if we still have anything that we write to at runtime in `db/`. Migrating `/var/lib/foreman/db` back to `/usr/share/foreman/db` is probably painful so I've never bothered but still something I wonder about every now and then.